### DISCUSSION TITLE
扩展 Shortcode - Echarts | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#495] [General]
# 扩展 Shortcode - Echarts | FixIt

echarts shortcode 使用 ECharts 库提供数据可视化的功能。

https://fixit.lruihao.cn/zh-cn/docs/content-management/shortcodes/extended/echarts/

**Comment by deepreadingnet** [comment]
使用echart shortcode，需要先下载echart.js到assets里吗

**Reply by Lruihao** [reply]
不需要
